### PR TITLE
Mn pool info related doc update

### PR DESCRIPTION
**Pull request by iamrz1**

### CODE REVIEW

**Review by dabasov — APPROVED** [review]
lgtm

**Review by peterlimg — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -1054,7 +1054,7 @@ Sample reformatted JSON output
 | Parameter   | Required | Description                                                  | Default | Valid Values |
 | ----------- | -------- | ------------------------------------------------------------ | ------- | ------------ |
 | `--id`      | Yes      | Node ID of a miner or sharder (get at `ls-miners` or `ls-sharders`) |         |              |
-| `--pool_id` | Yes      | Pool ID of a stake (get at `mn-info` or `mn-user-info`)      |         |              |
+| `--pool_id` | No       | Pool ID of a stake (get at `mn-info` or `mn-user-info`)      |         |              |
```
Comment (peterlimg) [inline README.md]: Seems this command does not have `pool_id` option anymore.

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Review by peterlimg — APPROVED** [review]
LGTM

**Pull request merged**